src/wp-includes/blocks.php

```diff
@@ -2420,6 +2421,48 @@ function build_query_vars_from_query_block( $block, $page ) {
 				}
 			}
 		}
+		if ( ! empty( $block->context['query']['format'] ) && is_array( $block->context['query']['format'] ) ) {
+			$formats   = $block->context['query']['format'];
+			$tax_query = array( 'relation' => 'OR' );
+
+			// The default post format, `standard`, is not stored in the database.
+			// If `standard` is part of the request, the query needs to exclude all post items that
+			// have a format assigned.
+			if ( in_array( 'standard', $formats, true ) ) {
+				$tax_query[] = array(
+					'taxonomy' => 'post_format',
+					'field'    => 'slug',
+					'terms'    => array(),
+					'operator' => 'NOT EXISTS',
+				);
+				// Remove the `standard` format, since it cannot be queried.
+				unset( $formats[ array_search( 'standard', $formats, true ) ] );
+			}
+			// Add any remaining formats to the tax query.
+			if ( ! empty( $formats ) ) {
```

Comment:
`! empty( $block->context['query']['format'] )` is already checked in line no. 2424. Should be removed.

Reply:
This is a check after the `standard` format is removed. See L2439.

Reply:
It is intended the check if there are any values left in case the 'standard' value was removed.
There is no need to run this code, if standard was the only value.

Reply:
Got it. Thanks!